src/nft/components/collection/Filters.css.ts

```diff
@@ -26,57 +26,67 @@ export const container = style([
 ])
 
 export const rowHover = style([
-  sprinkles({
-    borderRadius: '12',
-  }),
   {
     ':hover': {
-      background: themeVars.colors.backgroundSurface,
+      background: themeVars.colors.backgroundInteractive,
+      borderRadius: 12,
     },
   },
 ])
 
 export const rowHoverOpen = style([
-  sprinkles({
-    borderTopLeftRadius: '12',
-    borderTopRightRadius: '12',
-    borderBottomLeftRadius: '0',
-    borderBottomRightRadius: '0',
-  }),
   {
     ':hover': {
-      background: themeVars.colors.backgroundOutline,
+      background: themeVars.colors.backgroundInteractive,
     },
   },
 ])
 
 export const subRowHover = style({
   ':hover': {
-    background: themeVars.colors.backgroundOutline,
+    background: themeVars.colors.backgroundInteractive,
   },
 })
 
-export const detailsOpen = sprinkles({
-  background: 'backgroundModule',
-  overflow: 'hidden',
-  borderStyle: 'solid',
-  borderWidth: '1px',
-  borderColor: 'backgroundOutline',
-})
+export const detailsOpen = style([
+  sprinkles({
+    overflow: 'hidden',
+    marginTop: '8',
+    marginBottom: '8',
+  }),
+  {
+    borderTop: `1px solid ${themeVars.colors.backgroundOutline}`,
+  },
+])
 
-export const summaryOpen = sprinkles({
-  borderStyle: 'solid',
-  borderWidth: '1px',
-  borderColor: 'backgroundOutline',
-})
+export const detailsClosed = style([
+  detailsOpen,
+  sprinkles({
+    opacity: '0',
```

Comment:
I want this line to exist so component does not shoot up or down when toggling.  I change opacity when it is open